### PR TITLE
add courses to Arabic courses page

### PR DESCRIPTION
**Pull request by Zold1**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md)
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction)

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by Zold1** [comment]
don't merge now, I will add more courses

**Comment by Zold1** [comment]
I'm finished

**Comment by LuigiImVector** [comment]
Resources are okay, but some problems with the RTL must be resolved before merging, read this #6714

File: courses/free-courses-ar.md
Comment (LuigiImVector) [inline courses/free-courses-ar.md:58]:
```suggestion
* [Microprocessor 8086 & Assembly Language Course&rlm;](https://www.youtube.com/playlist?list=PLi0-RQZxQ8Fmwopq43StX61igOvXbFMQv) - Sherif Ezzat
```

File: courses/free-courses-ar.md
Comment (LuigiImVector) [inline courses/free-courses-ar.md:64]:
```suggestion
* [كورس بوتستراب كامل للمبتدئين | bootstrap 2021 tutorial for beginners&rlm;](https://www.youtube.com/playlist?list=PLknwEmKsW8OscL9GvjxwL7RYbcwwdIitk) - Abdelrahman Gamal
```

**Review by LuigiImVector — CHANGES_REQUESTED** [review]
You missed these two

File: courses/free-courses-ar.md
Comment (Zold1) [reply courses/free-courses-ar.md:58]: because the name of playlist is English not  Arabic

**Pull request merged**
